include/pfs/types.hpp

```diff
@@ -604,6 +604,22 @@ struct id_map
     size_t length = UINT32_MAX;
 };
 
+// Based on https://elixir.bootlin.com/linux/v5.19.10/source/net/ipv4/fib_trie.c#L2998
+struct route
+{
+    std::string iface;
+    ip destination;
+    ip gateway;
+    unsigned int flags;
```

Comment:
Have you considered going the extra mile and parsing the flags? You can find them [here](https://linux.die.net/man/8/route) and it should not be that hard to parse...

Reply:
I didn't see any reference to flag parsing on the library, can you give me an example from the library? not sure how a library user would like to use that

Reply:
After discussing that offline, I think we agreed to add that there's no point in doing it now.